packages/api-explorer/package.json

```diff
@@ -8,6 +8,7 @@
     "@readme/oas-extensions": "^4.8.1",
     "@readme/syntax-highlighter": "^4.8.4",
     "@readme/variable": "^4.8.1",
+    "api-explorer": "github:readmeio/api-explorer#beta/editor-hacking",
```

Comment:
Do we need this here? Package-ception!

Reply:
this is only here to get the `Content.jsx` working in the PR app! nixed it (tho this will break https://github.com/readmeio/readme/pull/1860 if we try to rebuild it!)